CHANGELOG.md

```diff
@@ -50,6 +50,7 @@
 
 #### [nidcpower] Unreleased
 - Added
+  - `Topic :: Scientific/Engineering :: Instrument Drivers` classifier
```

Comment:
This should either be added to the unreleased section for all of our APIs or for none of them.

Reply:
I'm not sure it's worth adding, given that there's no behavioral change.